wgpu-types/src/lib.rs

```diff
@@ -926,6 +930,11 @@ pub struct SamplerDescriptor<L> {
     pub lod_min_clamp: f32,
     pub lod_max_clamp: f32,
     pub compare: CompareFunction,
+    /// Anisotropic filtering extension must be enabled.
+    ///
+    /// Valid values are 1, 2, 4, 8, 16, and 32.
+    // TODO: implement device limits for anisotropy level
+    pub anisotropy_clamp: Option<u8>,
```

Comment:
We can't have `Option` in the `repr(C)` struct here, since it will make it no longer FFI-compatible. I know we agreed to change that semantics in #689 but this PR doesn't need to be that forcing function. Let's instead just have it as `u8` here and treat 0 and 1 as "disabled".